package_control/package_manager.py

```diff
@@ -2326,22 +2327,44 @@ def record_usage(self, params):
         params['sublime_platform'] = self.settings.get('platform')
         params['sublime_version'] = self.settings.get('version')
 
-        url = self.settings.get('submit_url', '') + '?' + urlencode(params)
+        # packagecontrol.io
+        url = self.settings.get('submit_url', '')
+        if url:
+            url += '?' + urlencode(params)
 
-        try:
-            result = http_get(url, self.settings, 'Error submitting usage information.')
-        except (DownloaderException) as e:
-            console_write(e)
-            return
+            try:
+                result = http_get(url, self.settings, 'Error submitting usage information.')
+            except (DownloaderException) as e:
+                console_write(e)
+                return
 
-        try:
-            result = json.loads(result.decode('utf-8'))
-            if result['result'] != 'success':
-                raise ValueError()
-        except (ValueError):
-            console_write(
-                '''
-                Error submitting usage information for %s
-                ''',
-                params['package']
-            )
+            try:
+                result = json.loads(result.decode('utf-8'))
+                if result['result'] != 'success':
+                    raise ValueError()
+            except (ValueError):
+                console_write(
+                    '''
+                    Error submitting usage information for %s
+                    ''',
+                    params['package']
+                )
+
+        # new stats
+        url = self.settings.get('submit_usage_url', '')
+        if url:
+            url += '?' + urlencode({"pkg": params["package"], "type": params["operation"]})
```

Comment:
Afaict, the new stat reporting includes strictly less information than the old one (only two parameters with a different name). Was there any particular reason why the old interface was not adopted, @kaste?  The only one I can think of would be reducing the amount of information that is transmitted to the absolute minimum for data protection, but especially things like the ST and PC version seem quite useful for determining the target audience. PC has historically been our only place to collect and provide this kind of information.

If the new endpoint used the same interface, we could simply make the `submit_url` an array and have it report to the two (or any number of) URLs with all the available data and then just ignore at the server side what we aren't interested in (yet).

Reply:
I think wbond overshared data.  We only wanted to collect data that we also "give back" to the user -- in the UI of the website.  That makes it transparent.  I think there must be a plan how we present the data.  And if that justifies the collection.  Data is always interesting.  We did not even use the update/removed event type as of yet. wbond just showed the ever increasing "install" count.

Reply:
As I did this, OS specific stats were dropped as "not-planned" UI wise.  Maybe the PC release cycle should allow some spontaneity.

Reply:
@FichteFoll send the data, we just drop unused data for now. But if you send it, we can still use it later as desired